frontend/src/libs/maps/icon-name-to-plain-svg.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactComponent as PlayIcon } from '#assets/icons//play.svg';
```

Comment:
```suggestion
import playIcon from '#assets/icons/play.svg';
```

Reply:
As we are moving to Icon component, so { ReactComponent as PlayIcon } is the only solution)